crates/nostr/src/nips/nip11.rs

```diff
@@ -69,6 +71,99 @@ pub struct RelayInformationDocument {
     pub software: Option<String>,
     /// Software version
     pub version: Option<String>,
+    /// Limitations imposed by the relay on clients
+    pub limitation: Option<Limitation>,
+    /// The relay's retention policies
+    pub retention: Option<Vec<Retention>>,
+    /// Country codes whose laws and policies may affect this relay
+    pub relay_countries: Option<Vec<String>>,
```

Comment:
Can you replace the `Option<Vec<T>>` with `Vec<T>` and add these pro-macros?

```rust
#[serde(skip_serializing_if = "Vec::is_empty")]
#[serde(default)]
pub relay_countries: Vec<String>,
```

The same in other pieces of code in this module.